python_bindings/tests/end-to-python-end-test/two_player_game_test.py

```diff
@@ -0,0 +1,572 @@
+-1         .[5].                  
+           .[!].                    
+           .....            
+"""
+
+
+def check_fifth_turn(engine: GameEngine, game, game_id) -> (int, SerializedGame):
+    turn_params = TurnParams(
+        pos=Position(x=-1, y=-1),
+        tile=game.current_tile,
+        meepleType=MeepleType.NormalMeeple,
+        side=Side.NoSide,
+        featureType=FeatureType.Monastery,
+    )
+
+    game_id, game = make_turn(engine, game, game_id, turn_params)
+    check_points(game, [4, 2])
+
+    return game_id, game
+
+
+"""
+// Two city edges not connected
+// player 2 places meeple(@) on the right city
+            -1    0    1    2    3
+            
+            
+                ..........          
+                ..........          
+1               --1--@-2..         
+                ./ \...|..          
+                |   |..|..          
+           ..|..|   |..|..          
+           ..|...\ /.!.|..          
+0          ..4----0----3..          
+           ..|............          
+           ..|............          
+           ..|..     -...-          
+           .[ ].      \./          
+-1         .[5].       6 @        
+           .[!].      /.\           
+           .....     -...-      
+"""
+
+
+def check_sixth_turn(engine: GameEngine, game, game_id) -> (int, SerializedGame):
+    turn_params = TurnParams(
+        pos=Position(x=1, y=-1),
+        tile=game.current_tile,
+        meepleType=MeepleType.NormalMeeple,
+        side=Side.Right,
+        featureType=FeatureType.City,
+    )
+
+    game_id, game = make_turn(engine, game, game_id, turn_params)
+    check_points(game, [4, 2])
+
+    return game_id, game
+
+
+"""
+// Two city edges not connected
+// player 1 places meeple (!) on the right city
+// playey 2 scores points for finished city
+            -1    0    1    2    3
+            
+            
+                ..........          
+                ..........          
+1               --1--@-2..         
+                ./ \...|..          
+                |   |..|..          
+           ..|..|   |..|..          
+           ..|...\ /.!.|..          
+0          ..4----0----3..          
+           ..|............          
+           ..|............          
+           ..|..     -...--...-     
+           .[ ].      \./  \./     
+-1         .[5].       6    7 !   
+           .[!].      /.\  /.\      
+           .....     -...--   - 
+"""
+
+
+def check_seventh_turn(engine: GameEngine, game, game_id) -> (int, SerializedGame):
+    turn_params = TurnParams(
+        pos=Position(x=2, y=-1),
+        tile=game.current_tile,
+        meepleType=MeepleType.NormalMeeple,
+        side=Side.Right,
+        featureType=FeatureType.City,
+    )
+
+    game_id, game = make_turn(engine, game, game_id, turn_params)
+    check_points(game, [4, 6])
+
+    return game_id, game
+
+
+"""
+// straight road
+// player 2 places meeple (@) on a bottom road
```

Comment:
```suggestion
// player 2 places meeple (@) on the road
```
there is only one road on this tile

Reply:
fixed